contrib/cirrus/setup.sh

```diff
@@ -19,7 +19,11 @@ msg "************************************************************"
 msg "Setting up runtime environment"
 msg "************************************************************"
 
-dnf -y install make automake gcc gcc-c++ kernel-devel
+dnf -y copr enable rhcontainerbot/podman-next
+dnf -y install make automake gcc gcc-c++ kernel-devel aardvark-dns
+
+# Remove netavark executable
+rm /usr/libexec/podman/netavark
```

Comment:
I think this will fail if the file doesn't exist: Use the `-f` argument please.  Also see https://github.com/containers/automation_images/pull/149

Reply:
yes

Reply:
I left a few comments on the automation_images PR. PTAL there